finetune/encoding/target_encoders.py

```diff
@@ -186,13 +186,19 @@ def target_labels(self):
 
 class SequenceLabelingEncoder(BaseEncoder):
 
-    def __init__(self, pad_token):
+    def __init__(self, pad_token, bio_tagging=False):
         self.classes_ = None
         self.pad_token = pad_token
         self.lookup = None
+        self.bio_tagging = bio_tagging
```

Comment:
I think in general it's preferable to factor this sort of think out into a different class that inherits some shared utils from the SequenceLabelingEncoder but doesn't change critical path code.

Reply:
Makes sense. I could see writing another class with a `transform` that closely matches the normal one, or one where we make a `super().transform()` call and modifies the labels after they've been tokenized and turned into IDs. Is there a preference? The former feels like it would duplicate a fair bit of code, while the latter seems a little bit hacky.

Reply:
Maybe this is about as clean as it gets... I'm just going to go ahead and approve, it does look like factoring out might be more trouble than it's worth.